common/io/Serial.cpp

```diff
@@ -161,6 +161,7 @@ bool OpenAndFlock(const std::string &path, int oflag, int *fd) {
     return false;
   }
 #else  // HAVE_FLOCK
```

Comment:
We don't normally bother commenting the else (and I personally think it makes it a bit more confusing as it's !HAVE_FLOCK).
```suggestion
#else
```